packages/codec/lib/index.ts

```diff
@@ -16,7 +16,147 @@ require("source-map-support/register");
 import * as Format from "@truffle/codec/format";
 export {
   /**
-   * Here's the decoder output format.
+   * # Codec Output Format
+   *
+   * ## Module information
+   *
+   * This module primarily defines TypeScript types for the output format
+   * used in results provided by packages
+   * `@truffle/decoder@^5.0.0` and `@truffle/codec@^0.1.0`.
+   *
+   * See below for complete listing or continue reading
+   * [Format information](#format-information) to learn about this format.
+   *
+   * ### How to import
+   *
+   * Import either as part of Codec or by itself:
+   *
+   * ```typescript
+   * // when importing entire Codec, use Codec.Format.*:
+   * import * as Codec from "@truffle/codec";
+   *
+   * // or import Format directly:
+   * import { Format } from "@truffle/codec";
+   * ```
+   *
+   * ![Example struct decoding](media://example-struct-decoding.png)
+   *
+   * ## Format information
```

Comment:
I just want to say, I like this introduction that you've written.  It's quite helpful.

Reply:
Thank you!